### DISCUSSION TITLE
5.0.0-alpha.3

### DISCUSSION THREAD

**Discussion by awmleer** [ant-design/ant-design-mobile#3860] [Announcements]
- 特性
  - InfiniteScroll 支持了自定义指示内容区域
  - Button 和 Tabs 组件支持了 data 属性和 aria 属性（试点验证）
- 其他
  - 更新了文档站点

---

迁移建议

无（本次发布不含 break change）

<hr /><em>This discussion was created from the release <a href='https://github.com/ant-design/ant-design-mobile/releases/tag/v5.0.0-alpha.3'>5.0.0-alpha.3</a>.</em>

**Comment by lyh-hub** [comment]
form表单input用form.setFieldsValue()受控会报错，只能用setState()吗

**Reply by awmleer** [reply]
可以给个复现的 demo 不～直接在 codepen 上写就行：https://codepen.io/pen/?template=zYwpMMd
